src/TreeContainer.js

```diff
@@ -107,6 +109,43 @@ class TreeContainer extends Component {
         };
     }
 
+    getNestedIds() {
+        const {
```

Comment:
Traverse all children for those with an id. Prune other components with `deepstate` flag set to true